docs/instruments/bmi.json

```diff
@@ -0,0 +1,532 @@
+{
+  "schema": {
+    "meta": {
+      "createdBy": "Zaliqa Rosli",
+      "createdOn": "2019-02-19",
+      "createdWith": "",
+      "defaultLanguage": "en-CA",
+      "derivedFrom": "",
+      "importedFrom": "bmi.linst",
```

Comment:
..., bmi.rules, bmi.score

Reply:
I don't think adding these are necessary. the instrument itself is the linst file. the use of *.rules is not currently reliable and the *.score contains calculations that accompany the instrument. if "importedFrom" points to a .linst instrument, it's up to the source instrument to show whether or not it's accompanied by a *.score or *.rules file i.e. they should be in the same directory as noted on the linst documentation (https://github.com/aces/Loris/wiki/Behavioural-Database)